test/e2e/node/pods.go

```diff
@@ -415,6 +415,160 @@ var _ = SIGDescribe("Pods Extended", func() {
 		})
 	})
 
+	ginkgo.Describe("Pod Generation", func() {
+		var podClient *e2epod.PodClient
+		ginkgo.BeforeEach(func() {
+			podClient = e2epod.NewPodClient(f)
+		})
+
+		ginkgo.It("pod generation should start at 1 and increment per update", func(ctx context.Context) {
+			ginkgo.By("creating the pod")
+			name := "pod-generation-" + string(uuid.NewUUID())
+			value := strconv.Itoa(time.Now().Nanosecond())
+			pod := e2epod.NewAgnhostPod(f.Namespace.Name, name, nil, nil, nil)
+			pod.ObjectMeta.Labels = map[string]string{
+				"time": value,
+			}
+			pod.Spec.InitContainers = []v1.Container{{
+				Name:    "init-container",
+				Image:   "busybox:1.28",
+				Command: []string{"sh", "-c", "exit 0"},
+			}}
```

Comment:
nit: seems mostly unnecessary? Can you just use a simple pause pod instead? Maybe we should add a new helper for that

Reply:
turns out a helper for this already exists, so I updated this to use that. I changed this so that it starts out as busybox and gets updated to pause later (as part of checking that changing the container image bumps generation). But happy to change up the images if you prefer